go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/mattermost/mattermost-plugin-autolink
 go 1.12
 
 require (
+	github.com/gorilla/mux v1.7.3
```

Comment:
Do we _really_ need it?

Reply:
No. But it does everything, is well tested, and is fast to use.